components/event-bus/cmd/event-publish-service/application/application_test.go

```diff
@@ -90,32 +90,6 @@ func Test_PublishV1WithSourceIdInHeader_ShouldSucceed(t *testing.T) {
 	assert.Equal(t, testV1.TestEventID, publishResponse.EventID)
 }
 
-func Test_PublishV1WithChannelNameGreaterThanMaxChannelLength_ShouldFail(t *testing.T) {
```

Comment:
Why are we removing this test? Shouldn't this be valid still?

Reply:
I believe this is not valid anymore since we are not validating the length of the channel now.